QuizApp/DataLayer/API/NetworkServiceProtocol.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NetworkServiceProtocol.swift
+//  QuizApp
+//
+//  Created by Matej Butkovic on 14.05.2021..
+//
+
+import Foundation
+
+protocol NetworkServiceProtocol {
```

Comment:
This protocol should implement all three methods you've created above so the presenters using this network service don't know the exact implementation (NetworkService class) but only a protocol.